src/main/resources/application.conf

```diff
@@ -198,6 +198,22 @@ mantis {
     transaction-timeout = 2.minutes
   }
 
+  consensus {
+    # FIXME this was moved here from the `mining` section
+    active-timeout = 5 seconds
```

Comment:
OK, `rpc` seems better. I will make this change in a subsequent PR (along with the change below about `mining` -> `ethash`), since more refactoring is in the way.